webpack.extras.ts

```diff
@@ -0,0 +1,32 @@
+const loaderUtils = require('loader-utils');
+const path = require('path');
+
+const objExtend = function(args, obj) {
+  args = Array.prototype.slice.call(args);
+  const _a = args.slice(1);
+  _a.unshift(Object.assign(obj, args[0]));
+  return _a;
+};
+
+
+module.exports = {
+  load: function(resourcePath, args) {
+    resourcePath = `/${resourcePath}`;
+    const root = path.resolve(__dirname, 'core/templates/dev/head');
+    const argsExpr = args ? '(' + objExtend + ')' + '(arguments, ' +
+      JSON.stringify(args) + ')' : 'arguments';
+
+    return 'require(' + JSON.stringify(loaderUtils.urlToRequest(
+      resourcePath, root)) + ').apply(null,' + argsExpr + ')';
+  },
+
+  loadExtensions: function(resourcePath, args) {
+    resourcePath = `/${resourcePath}`;
+    const root = path.resolve(__dirname, 'extensions');
+    const argsExpr = args ? '(' + objExtend + ')' + '(arguments, ' +
+      JSON.stringify(args) + ')' : 'arguments';
+
+    return 'require(' + JSON.stringify(loaderUtils.urlToRequest(
+      resourcePath, root)) + ').apply(null,' + argsExpr + ')';
+  }
```

Comment:
Or maybe you can also add `loadFooterLibs` to load the `'pages/footer_js_libs.html'` and remvoe the pure `load` macro altogether.

Reply:
Could you point me to the line where this happens?

Reply:
@Jamesjay4199 Missing response from you.

Reply:
@vojtechjelinek, this can be seen [here](https://github.com/oppia/oppia/blob/c0bfd30f149b22fc497206c0d1356b8455d37fe4/core/templates/dev/head/pages/footer_js_libs.html#L1) and [here](https://github.com/oppia/oppia/blob/c0bfd30f149b22fc497206c0d1356b8455d37fe4/core/templates/dev/head/pages/header_css_libs.html#L3)